### PR TITLE
Milestone #3 - Delete and sort

### PR DESCRIPTION
**Pull request by GuilhermeGiachellin**
- Created a sorting algorithm for the index.
- Delete all and delete individual tasks that are checked.

### CODE REVIEW

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hello @GuilhermeGiachellin** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

- The trash icon is not functional, please modify the` const deletTrash` to be funcional.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/delete.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      if (tasks[index].complete === true) {
+        e.parentNode.parentNode.remove();
+        tasks.splice(index, 1);
+        localStorage.setItem('tasks', JSON.stringify(tasks));
+        component();
+        window.location.reload();
+      }
+    });
+  });
+};
```
Comment (dasileker) [inline src/delete.js]: - [x] you can append the trash to make it work, see below the trash icon is not working 👇

https://user-images.githubusercontent.com/61562626/126048941-7d16d806-caba-4673-84ea-6281ecd1c2fd.mp4

.

Comment (GuilhermeGiachellin) [reply src/delete.js]: Thank you! I thought that only checked tasks should be deleted, but I saw the video and I was wrong. So I changed as you requested!

**Review by abruzy — APPROVED** [review]
```markdown
Hi @GuilhermeGiachellin 👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**